$NetBSD: patch-ab,v 1.1.1.1 2009/02/23 19:32:01 minskim Exp $

Make sure teTeX3 compiles even if floor is defined by the OS (which is
the case on Mac OS 10.4 at least)

--- special.c.orig	2008-05-03 12:45:10.000000000 -0700
+++ special.c
@@ -181,7 +181,9 @@ static Boolean parse_color (const char *
 #define	MAX_PEN_SIZE	7	/* Max pixels of pen width */
 #define	TWOPI		(3.14159265359 * 2.0)
 
+#ifndef floor
 extern double floor(double);
+#endif
 #define	rint(x)	floor((x) + 0.5)
 
 static int xx[MAXPOINTS], yy[MAXPOINTS];	/* Path in milli-inches */
